java/com/engflow/notificationqueue/Client.java

```diff
@@ -0,0 +1,206 @@
+package com.engflow.notificationqueue;
```

Comment:
Before submitting, make sure to format java files with our formatter. This repo doesn't have a pre-commit hook for it, but you can still run the tool from here:

```
java -jar ../engflow/third_party/google-java-format/google-java-format-1.15.0-all-deps.jar -i java/com/engflow/example/Example.java
```